plugins/osdn/ovs/plugin.go

```diff
@@ -34,7 +34,7 @@ func MultiTenantPluginName() string {
 func CreatePlugin(registry *osdn.Registry, multitenant bool, hostname string, selfIP string) (api.OsdnPlugin, api.FilteringEndpointsConfigHandler, error) {
 	plugin := &ovsPlugin{multitenant: multitenant}
 
-	err := plugin.BaseInit(registry, plugin, hostname, selfIP)
+	err := plugin.BaseInit(registry, plugin, multitenant, hostname, selfIP)
 	if err != nil {
 		return nil, nil, err
 	}
```

Comment:
Yeah, it's dumb, but I didn't want to log two separate lines... (We can't log hostname and selfIP from here because in some cases they get computed by BaseInit(), and the computed IP isn't stored in a public struct member, so we can't access it after BaseInit() returns either. So we have to log that from there.) But I could change that.